tests/serializers/test_fraction.py

```diff
@@ -0,0 +1,57 @@
+from fractions import Fraction
+
+import pytest
+
+from pydantic_core import SchemaSerializer, core_schema
+
+
+def test_fraction():
+    v = SchemaSerializer(core_schema.fraction_schema())
+    assert v.to_python(Fraction('123.456')) == Fraction('123.456')
+
+    assert v.to_python(Fraction('123.456'), mode='json') == '123.456'
+    assert v.to_json(Fraction('123.456')) == b'"123.456"'
+
+    assert v.to_python(Fraction('123456789123456789123456789.123456789123456789123456789')) == Fraction(
+        '123456789123456789123456789.123456789123456789123456789'
+    )
+    assert (
+        v.to_json(Fraction('123456789123456789123456789.123456789123456789123456789'))
+        == b'"123456789123456789123456789.123456789123456789123456789"'
+    )
+
+    # with pytest.warns(UserWarning, match='Expected `fraction` but got `int` - serialized value may not be as expected'):
+    #     assert v.to_python(123, mode='json') == 123
+
+    # with pytest.warns(UserWarning, match='Expected `fraction` but got `int` - serialized value may not be as expected'):
+    #     assert v.to_json(123) == b'123'
```

Comment:
Are this tests relevant for the Fraction? I suppose they are not, because `serialized value may not be as expected` should be as expected